### DISCUSSION TITLE
[Feature] Adding to multiple albums at one time

### DISCUSSION THREAD

**Discussion by jfreak53** [immich-app/immich#11259] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be beneficial if you could select a bunch of images and add to multiple libraries at one time. For instance I have a fishing album, a kids album for family sharing, and a family album for my wife and myself. If I go fishing with the kids it would be nice if I could select all the pics in question and add to multiple albums at once since they go in all 3 albums.

Currently I have to do it 3 times, once per album. I envision something like when I select multiple images, long holding an album under add to, selects it with check, then I can add to multiple albums at once.

Not sure if web does this already, I know mobile doesnt I just tried.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by ByteSizedMarius** [comment]
I'd be interested in working on this for mobile, but I want to make sure it's something the maintainers actually want before I put time into it.

Also does this project require feature parity between mobile and web? Like, would web need the same thing implemented before this could be merged?

**Comment by bo0tzz** [comment]
#2686
